src/app/src/pages/ForgotPassword.js

```diff
@@ -0,0 +1,88 @@
+import { useState } from 'react';
+import { Box, Button, Input, Text, VStack } from '@chakra-ui/react';
+import { useNavigate } from 'react-router-dom';
+
+import { API } from '../api';
+import { API_URLS } from '../constants';
+
+export default function ForgotPassword() {
+    const [email, setEmail] = useState('');
+    const [success, setSuccess] = useState(false);
+    const [errorDetail, setErrorDetail] = useState('');
+    const navigate = useNavigate();
+
+    const formatApiError = apiErrorData => {
+        var errorDetail = '';
+        Object.values(apiErrorData).forEach(val => (errorDetail += `${val}`));
+        return errorDetail || 'An unknown error occurred.';
+    };
+
+    const forgotPasswordRequest = () => {
+        API.post(API_URLS.FORGOT, { email })
+            .then(() => {
+                setSuccess(true);
+                setErrorDetail('');
+            })
+            .catch(apiError => {
+                setSuccess(false);
+                setErrorDetail(formatApiError(apiError.response?.data));
+            });
+    };
+
+    return (
+        <VStack bg='gray.100' h='100vh' spacing={5}>
+            <Box h='150px'></Box>
```

Comment:
Consider using the same wrapper settings as in Login:

https://github.com/azavea/iow-boundary-tool/blob/1533a3594be4abac9a06b346ee3d7bea9e5c3065/src/app/src/pages/Login.js#L43-L44

```suggestion
        <VStack bg='gray.50' h='100vh' spacing={5}>
            <Box h='20%'></Box>
```

Can this be extracted into something common so it doesn't need to be changed in three places?

Reply:
Thanks, I was working ahead on this PR and forgot to keep it up to date with the final design changes in #77. 👍🏻